helm/tweek/templates/gateway.yaml

```diff
@@ -0,0 +1,107 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: tweek-gateway-{{ .Release.Name }}
+spec:
+  replicas: {{ .Values.gatewayReplicaCount }}
+  template:
+    metadata:
+      annotations:
+        checksum/config: {{ include (print $.Template.BasePath "/gateway-configmap.yaml") . | sha256sum }}
```

Comment:
What's the checksum config?

Reply:
It's a trick to reload the gateway pod every time the configmap changes.